watch.d/SRS_GRIDDED_OC.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Maybe we should call it `SRS_OC_GRIDDED` and also rename `SRS_GRIDDED_SST` to `SRS_SST_GRIDDED`.